app/javascript/react/components/molecules/FixedStreamStationHeader/atoms/StationActionButtons/StationActionButtons.tsx

```diff
@@ -24,7 +24,7 @@ const StationActionButtons = () => {
       </S.MobileButtons>
       <S.DesktopButtons>
         <Button
-          onClick={() => {}}
+          onClick={copyCurrentURL}
           aria-label={t("calendarHeader.altShareLink")}
         >
           {t("calendarHeader.copyLink")} <img src={copyLink} />
```

Comment:
We can also add to the image `alt` attribute. e.g. "Copy link" or something similar.